invoker/function-maven-plugin/src/main/java/com/google/cloud/functions/plugin/RunFunction.java

```diff
@@ -1,17 +1,14 @@
 package com.google.cloud.functions.plugin;
 
 import com.google.cloud.functions.invoker.runner.Invoker;
+import org.apache.maven.plugin.AbstractMojo;
+import org.apache.maven.plugin.MojoExecutionException;
+import org.apache.maven.plugins.annotations.*;
```

Comment:
Would you mind putting these imports back the way they were? Google style requires imports to be sorted lexicographically with no blank lines and no wildcard imports.